<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="14" failed="1" total="23" passed="8">
  <reporter-output>
  </reporter-output>
  <suite name="com.force.raidennet.api.ec2.impl.EC2ApiIntegrationTest" duration-ms="467205" started-at="2014-10-03T00:43:13Z" finished-at="2014-10-03T00:51:01Z">
    <groups>
      <group name="EC2ApiIntegrationTest.tags">
        <method signature="EC2ApiIntegrationTest.testCreateTags()[pri:0, instance:com.force.raidennet.api.ec2.impl.EC2ApiIntegrationTest@348fb08f]" name="testCreateTags" class="com.force.raidennet.api.ec2.impl.EC2ApiIntegrationTest"/>
        <method signature="EC2ApiIntegrationTest.testDeleteTags()[pri:0, instance:com.force.raidennet.api.ec2.impl.EC2ApiIntegrationTest@348fb08f]" name="testDeleteTags" class="com.force.raidennet.api.ec2.impl.EC2ApiIntegrationTest"/>
      </group>
    </groups>
    <test name="Command line test" duration-ms="467205" started-at="2014-10-03T00:43:13Z" finished-at="2014-10-03T00:51:01Z">
      <class name="com.force.raidennet.api.ec2.impl.EC2ApiIntegrationTest">
        <test-method status="FAIL" signature="tearDownClass()[pri:0, instance:com.force.raidennet.api.ec2.impl.EC2ApiIntegrationTest@348fb08f]" name="tearDownClass" is-config="true" duration-ms="0" started-at="2014-10-03T00:50:10Z" finished-at="2014-10-03T00:50:10Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Unable to fully clean up after integration tests. Please refer to log for errors: [java.lang.AssertionError: Unable to terminate instance i-000000000000005. You will have to manually delete this.] expected:<true> but was:<false>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Unable to fully clean up after integration tests. Please refer to log for errors: [java.lang.AssertionError: Unable to terminate instance i-000000000000005. You will have to manually delete this.] expected:<true> but was:<false>
	at org.testng.Assert.fail(Assert.java:89)
	at org.testng.Assert.failNotEquals(Assert.java:489)
	at org.testng.Assert.assertTrue(Assert.java:37)
	at com.force.raidennet.api.ec2.impl.EC2ApiIntegrationTest.tearDownClass(EC2ApiIntegrationTest.java:178)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:543)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:212)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:225)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:114)
	at org.testng.TestRunner.privateRun(TestRunner.java:758)
	at org.testng.TestRunner.run(TestRunner.java:613)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:87)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1137)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1062)
	at org.testng.TestNG.run(TestNG.java:974)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:115)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:104)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
      </class>
    </test>
  </suite>
</testng-results>
